File: apps/s-2-admin/src/api/feedback/FeedbackWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { DateTimeFilter } from "../../util/DateTimeFilter";
import { UserWhereUniqueInput } from "../user/UserWhereUniqueInput";
import { StringFilter } from "../../util/StringFilter";
import { StringNullableFilter } from "../../util/StringNullableFilter";

export type FeedbackWhereInput = {
  id?: IntFilter;
  date?: DateTimeFilter;
  user?: UserWhereUniqueInput;
  rating?: StringFilter;
  comment?: StringNullableFilter;
};

File: apps/cactus-demo-dotnet/src/APIs/Feedback/Dtos/FeedbackWhereInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class FeedbackWhereInput
{
    public int? Id { get; set; }

    public DateTime? Date { get; set; }

    public int? User { get; set; }

    public string? Rating { get; set; }

    public string? Comment { get; set; }
}
